docker-compose/high-availability/prometheus1.yml

```diff
@@ -8,7 +8,8 @@ global:
   # Attach these labels to any time series or alerts when communicating with
   # external systems (federation, remote storage, Alertmanager).
   external_labels:
-      monitor: 'codelab-monitor'
+      cluster: 'monitoring-cluster'
+      __replica__: 'promscale_deploy_ha-promscale-connector1-1'
```

Comment:
the name looks a bit weird. Can't we simply name `promscale-replica-1` and similarly for the other one?

Reply:
Its weird because thats the name of the container that docker compose generates. It simplifies the test for checking which replica is the current leader. Changing into something simpler would complicate the testing a bit. 

IMHO don't think its worth the trouble in this case.